app/models/miq_schedule_worker/jobs.rb

```diff
@@ -143,6 +143,12 @@ def metric_purge_all_timer
     end
   end
 
+  def database_maintenance_reindex_timer
+    %w(VmdbMetric MiqQueue MiqWorker).each do |class_name|
```

Comment:
I wonder if it would be easier to keep this list in a setting then have the class implement just the `reindex_table` method.

Then we could just add the `.reindex` method [here](https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/ar_base.rb) and avoid the mixin entirely.

What do you think?